drafts/draft-ietf-ippm-ioam-data-integrity.xml

```diff
@@ -734,7 +740,7 @@
         <section title="Replay protection">
         <t>The nonce makes a signature chain unique but does not necessarily
         prevent replay attacks. To enable replay protection, both the encap node
-        and the validator MUST synchronize on a unique nonce, e.g., based on a
+        and the validator SHOULD synchronize on a unique nonce, e.g., based on a
```

Comment:
Why the change to SHOULD? 

We might rather say that "encap node and validator MUST use a common, unique nonce" - that avoids the statement about synchronization.